wyvern/components/models/modelbit_component.py

```diff
@@ -49,7 +49,7 @@ def __init__(
         self._auth_token = auth_token or self.AUTH_TOKEN
         self._modelbit_url = url or self.URL
         self.headers = {
-            "Authorization": f"Bearer {self._auth_token}",
```

Comment:
according to modelbit documentation: https://doc.modelbit.com/deployments/rest-api/api-keys

`  -H "Authorization: YOUR_API_KEY"`